consai_game/consai_game/world_model/world_model_provider_node.py

```diff
@@ -79,6 +81,10 @@ def __init__(self, update_hz: float = 10, team_is_yellow: bool = False, goalie_i
         self.sub_param_control = self.create_subscription(
             String, "consai_param/control", self.callback_param_control, qos_profile
         )
+        # motion_commandのsubscriber
+        self._sub_motion_command = self.create_subscription(
+            MotionCommandArray, "motion_commands", self.callback_desired_pose, 10
```

Comment:
motion_commandsを受け取る関数なので、名前を変えたいです。
```suggestion
            MotionCommandArray, "motion_commands", self.callback_motion_commands, 10
```

Reply:
meme: finish